resources/views/pages/password/reset.twig

```diff
@@ -9,7 +9,7 @@
         <h1>{{ __('Password recovery') }}</h1>
 
         {% for message in errors|default([]) %}
-            {{ m.alert(__(message), 'danger') }}
+            {{ m.alert(message, 'danger') }}
```

Comment:
Same as `resources/views/pages/login.twig` (this needs a cleanup anyway to prevent code duplication..)

Reply:
(see comment below)